### PR TITLE
Write createDataChannel. Fixes #300

### LINKED ISSUE #300: Add Section for createDataChannel
JSEP currently has this:

4.1.4.  createDataChannel

   [TODO]

We need this text before WGLC, so it's being added to the punchlist.

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -1165,7 +1165,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 </section>
 
                 <section title="createDataChannel" anchor="sec.createDataChannel">
-                  <t>[TODO]</t>
+                  <t>
+                    The createDataChannel method creates a data channel and
+                    attaches it to the PeerConnection. If no data channel exists
+                    on this PeerConnection, then creating a data channel will
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: perhaps: 

    If no data channel currently exists for this PeerConnection, a new offer/answer   
exchange is required.

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -1165,7 +1165,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                    also require a new offer/answer exchange. All data channels on
+                    a given PeerConnection share the same SCTP/DTLS association
+                    and therefore the same m= section, so subsequent data channels
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: 'subsequent creation of data channels does not have any impact on the JSEP state'

Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: I think you may also want to expand this to be super clear that the only processing required from JSEP is the creation of the SCTP/DTLS association, i.e. the supplied parameters to createDataChannel are not used by JSEP.

**Comment by ekr** [comment]
@juberti See above

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -1165,7 +1165,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 </section>
 
                 <section title="createDataChannel" anchor="sec.createDataChannel">
-                  <t>[TODO]</t>
+                  <t>
+                    The createDataChannel method creates a data
+                    channel and attaches it to the PeerConnection. If no data channel
+                    currently exists for this PeerConnection, then a new offer/anser
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: typo: answer

File: draft-ietf-rtcweb-jsep.xml [hunk 4]
```diff
@@ -1165,7 +1165,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
+                    exchange is required. All data channels on a given PeerConnection
+                    share the same SCTP/DTLS association and therefore the same m=
+                    section, so subsequent creation of data channels does not have any
+                    impact on the JSEP state. The createDataChannel method also includes
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Suggest new paragraph for the final sentence

Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Didn't see this in the latest commit

**Comment by juberti** [comment]
LGTM when outstanding comment addressed

**Pull request merged**